### DISCUSSION TITLE
Disable cycle though thirds

### DISCUSSION THREAD

**Discussion by Mr-iX** [rxhanson/Rectangle#519] [Q&A]
Is it possible to disable that Rectangle cycles a window throw the first, second and third 1/3 of the screen when pressing CONTROL + ALT + D multiple times?

**Comment by rxhanson** [comment]
Right now this is can be disabled only by disabling cycling behavior for all window actions.

To do this, execute this terminal command and restart the app:

`defaults write com.knollsoft.Rectangle subsequentExecutionMode -int 2`
